### DISCUSSION TITLE
```NOT AN ISSUE ```  JUST FOR HELP

### DISCUSSION THREAD

**Discussion by manishgautammg3994** [serverpod/serverpod#533] [Ideas]
[Get_Cli](https://github.com/jonataslaw/get_cli[) Source Code can help you for all those ```issues``` #500 also [Get Server](https://github.com/jonataslaw/get_server) Source Code can resolve your WebServer with Widgets ```problem``` , I wish You will Explore them Too ,
## Thanks in advance

**Comment by yahu1031** [comment]
@manishgautammg3994 C'mon bruh, I(Personally) never wanna see getX practices here. Serverpod is really good in its way. And moreover, we don't have such a requirement for the enhancement.

**Comment by tusharbhambere** [comment]
We don't want People start Connecting/Comparing Serverpod with Getx. Serverpod  tools is  enough
